backend/framework/bin/exposure_generator.py

```diff
@@ -19,8 +19,9 @@
 
 spectro_path = os.path.normpath(cfg.get("namespace", "desi_spectro_data"))
 spectro_redux = os.path.normpath(cfg.get("namespace", "desi_spectro_redux"))
-fiberflat_path = os.path.join(spectro_redux, "calib2d")
-psfboot_path = os.path.join(fiberflat_path, "psf")
+base_exposures_path = cfg.get("namespace", "base_exposures_path")
+fiberflat_path = os.path.join(base_exposures_path, "calib2d")
```

Comment:
Please use `desispec.io.findfile(filetype, night, expid, camera)` to identify the directories where files should go.  This centralizes the choices for file locations to simplify updated.  e.g. the `calib2d` directory was refactored in March (desihub/desispec#545) to put the per-exposure fiberflats into `exposures/NIGHT/EXPID/fiberflat-*.fits` and the combined nightly fiberflat into `calibnight/NIGHT/` .  i.e. the `calib2d` directory no longer exists.